src/main/java/de/retest/recheck/RecheckImpl.java

```diff
@@ -77,7 +81,12 @@ public RecheckImpl( final RecheckOptions options ) {
 		fileNamerStrategy = options.getFileNamerStrategy();
 		suiteName = options.getSuiteName();
 		suite = SuiteReplayResultProvider.getInstance().getSuite( suiteName );
-		ignoreApplier = RecheckIgnoreUtil.loadRecheckIgnore();
+
+		final GlobalIgnoreApplier globalIgnoreApplier = RecheckIgnoreUtil.loadRecheckIgnore();
+		final GlobalIgnoreApplier suiteIgnoreApplier = RecheckIgnoreUtil.loadRecheckSuiteIgnore( getSuitePath() );
+
+		ignoreApplier = new CompoundFilter( Lists.newArrayList( globalIgnoreApplier, suiteIgnoreApplier ) );
```

Comment:
IMHO it is good to not rely on Guava if there is similar JDK method:

```suggestion
		ignoreApplier = new CompoundFilter( Arrays.asList( globalIgnoreApplier, suiteIgnoreApplier ) );
```